### PR TITLE
Resoved problems

### PR DESCRIPTION
**Pull request by Bravax**

### CODE REVIEW

**Review by ItsMatDev — APPROVED** [review]

File: frontend/src/components/HomePage/ContainerStoryRules/ContainerStoryRules.scss [hunk 1]
```diff
@@ -12,21 +12,27 @@
 
   &__playButton {
     width: 15vw;
-    font-size: 2rem;
+    font-size: 2vw;
     border-color: $secondaryColor;
     background-color: $secondaryColor;
-    height: 5rem;
+    height: 3.5vw;
```
Comment (Yoann-Fortin) [inline frontend/src/components/HomePage/ContainerStoryRules/ContainerStoryRules.scss]: No vw on height

File: frontend/src/components/LevelPage/ReturnGameMenu/ReturnGameMenu.scss [hunk 2]
```diff
@@ -1,9 +1,7 @@
 @import "../../../variables.scss";
 
 .backMainMenuButton {
-  position: absolute;
-  right: 0;
-  margin-top: 2vw;
+  top: 2vw;
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/ReturnGameMenu/ReturnGameMenu.scss]: No vw on height

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

File: frontend/src/components/LevelPage/ReturnGameMenu/ReturnGameMenu.scss [hunk 3]
```diff
@@ -1,6 +1,8 @@
 @import "../../../variables.scss";
 
 .backMainMenuButton {
+  position: absolute;
+  right: 0;
   top: 2vw;
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/ReturnGameMenu/ReturnGameMenu.scss]: No vw on height

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

File: frontend/src/components/HomePage/ContainerStoryRules/ContainerStoryRules.scss (hunk 1 above)
Comment (Bravax) [reply frontend/src/components/HomePage/ContainerStoryRules/ContainerStoryRules.scss]: Rectified

File: frontend/src/components/LevelPage/ReturnGameMenu/ReturnGameMenu.scss (hunk 2 above)
Comment (Bravax) [reply frontend/src/components/LevelPage/ReturnGameMenu/ReturnGameMenu.scss]: Rectified

File: frontend/src/components/LevelPage/ReturnGameMenu/ReturnGameMenu.scss (hunk 3 above)
Comment (Bravax) [reply frontend/src/components/LevelPage/ReturnGameMenu/ReturnGameMenu.scss]: Rectified

Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/ReturnGameMenu/ReturnGameMenu.scss]: No vw on height

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**